vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/create/ProjectInformationDialog.java

```diff
@@ -181,6 +207,12 @@ public void reset() {
         .forEach(component -> ((HasValue<?, ?>) component).clear());
     formLayout.getChildren().filter(component -> component instanceof HasValidation)
         .forEach(component -> ((HasValidation) component).setInvalid(false));
+
+    // reset components in sub-layout
+    codeAndTitleLayout.getChildren().filter(component -> component instanceof HasValue<?, ?>)
+        .forEach(component -> ((HasValue<?, ?>) component).clear());
+    codeAndTitleLayout.getChildren().filter(component -> component instanceof HasValidation)
+        .forEach(component -> ((HasValidation) component).setInvalid(false));
```

Comment:
NIT since this is repetitive code maybe a function can facilitate the reset of (sub)components

Reply:
good idea, changed